src/emsdevice.cpp

```diff
@@ -143,27 +143,27 @@ std::string EMSdevice::device_type_name() const {
         break;
 
     case DeviceType::SOLAR:
```

Comment:
I may refactor this code since its the same as the other function, only the first letter is capitalized